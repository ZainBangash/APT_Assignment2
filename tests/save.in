1
TANIA
ZAIN
save TEST