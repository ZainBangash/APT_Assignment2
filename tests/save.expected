Welcome To Scrabble!
------------------- 
Menu
----
1. New Game
2. Load Game        
3. Credits
4. Quit
1

Starting a New Game

Enter a name for player 1 (uppercase characters only)
TANIA

Enter a name for player 2 (uppercase characters only)
ZAIN


Player 1: TANIA, 0
Player 2: ZAIN, 0

Lets Play!

     1   2   3   4   5   6   7   8   9   10  11  12  13  14  15
  --------------------------------------------------------------
 A |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 B |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 C |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 D |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 E |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 F |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 G |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 H |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 I |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 J |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 K |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 L |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 M |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 N |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
 O |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
J-8 E-1 A-1 M-3 Y-4 L-1 Q-10
Enter command: save TEST


Game saved successfully

Welcome To Scrabble!
-------------------
Menu
----
1. New Game
2. Load Game
3. Credits
4. Quit